packages/api-clients/open-api/bffApi.yml

```diff
@@ -13537,6 +13537,8 @@ components:
           $ref: "#/components/schemas/CompactDescriptor"
         isDelegable:
           type: boolean
+        isClientAccessDelegable:
+          type: boolean
```

Comment:
See same comment I left here in the PR for `isDelegable`: https://github.com/pagopa/interop-be-monorepo/pull/1188#discussion_r1843638162